### PR TITLE
Fix bugs - joinG no error message for duplicate person - expand task's name to new line

### PR DESCRIPTION
**Pull request by hungkhoaitay**

### CODE REVIEW

File: src/main/resources/view/TaskCard.fxml [hunk 1]
```diff
@@ -23,7 +23,7 @@
             <Region fx:constant="USE_PREF_SIZE" />
           </minWidth>
         </Label>
-        <Label fx:id="description" text="\$first" styleClass="cell_big_label" />
+        <Label fx:id="description" text="\$first" styleClass="cell_big_label" maxHeight="50" wrapText="true" />
```
Comment (jayasting98) [inline src/main/resources/view/TaskCard.fxml:26]: I'm not sure if `maxHeight` is necessary. I think it causes it to stop wrapping when the max height is reached.

Without `maxHeight`:
image

With `maxHeight`:
image

Comment (hungkhoaitay) [reply src/main/resources/view/TaskCard.fxml:26]: Yes @jayasting98, I thought we are going to limit the Task's name to two lines, should I make it four lines? @wonyeji

Comment (hungkhoaitay) [reply src/main/resources/view/TaskCard.fxml:26]: @asherhy @thanwinnikki

**Review by wonyeji — APPROVED** [review]
LGTM.

**Pull request merged**